atoti-query-analyser/src/components/Summary/Summary.tsx

```diff
@@ -200,6 +201,96 @@ function MapView<K, V>({
   );
 }
 
+type ComponentTimings = Readonly<{
+  database: number;
+  engine: number;
+  network: number;
+  providers: number;
+}>;
+
+const computeComponentTimings = (graph: RetrievalGraph): ComponentTimings => {
```

Comment:
The sum of execution times may be confusing sometimes. It would be also interesting to see wall clock time (maybe as a time diagram with overlapping rectangles). What do you think?

Reply:
After a live discussion with @ogorodnikoff2012 , we concluded that it was a matter of taste. Some may be interested in knowing the exact time, while a rounding is also ok to show elements in the same order.
We will start this way and switch to something more fancy if needed.